stage1/init/kvm/network.go

```diff
@@ -158,7 +158,9 @@ func GenerateNetworkInterfaceUnits(unitsPath string, netDescriptions []netDescri
 			return errwrap.Wrap(fmt.Errorf("failed to create network unit file %q", unitName), err)
 		}
 
-		rlog.Printf("network unit created: %q in %q (iface=%q, addr=%q)", unitName, unitsPath, ifName, address)
+		if debug {
```

Comment:
At the entry of package? But this is dependent from cmdline setting. I don't exactly follow where/how this should be controlled.
Could you prepare come gist, which would show how this should be done there?